feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -263,8 +267,25 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
     }
   }
 
-  andThen(_options: any) {
-    //navigate to other pages
+  protected handleDeepLinkParamsAfterStartSession(
+    parameters: AsmDeepLinkParameters
+  ) {
+    if (parameters.orderId) {
+      // Navigate to order details
+      this.routingService.go({
+        cxRoute: 'orderDetails',
+        params: { code: parameters.orderId },
+      });
+    } else if (parameters.ticketId) {
+      // Navigate to support ticket details
+      this.routingService.go({
+        cxRoute: 'supportTicketDetails',
+        params: { ticketCode: parameters.ticketId },
+      });
+    } else if (parameters.savedCartId) {
```

Comment:
check https://jira.tools.sap/browse/CXEC-22589 here only `cartId` and seems need to check
cartType = this.asmComponentService.getSearchParameter('cartType');